src/wp-includes/css/custom-properties.css

```diff
@@ -2,6 +2,7 @@ body {
 	--wp-admin--theme--primary: #2271b1;
 	--wp-admin--theme--primary--contrast: #f6f7f7;
 	--wp-admin--theme--primary--step-10: #1a5686;
+	--wp-admin--theme--primary--step-X: #0a4b78;
```

Comment:
Note this value `--wp-admin--theme--primary--step-X` used on lines 168 & 169. I am not sure how to calculate what step number this value would be but it's in the color palette (https://codepen.io/ryelle/full/WNGVEjw) as `blue 70` and primary color is `blue 50`, or if it's even what we mean by steps here. I don't see step-10 defined in the color palette at all, so looking for guidance here. Maybe I should have just added it as a custom property specific to this placement?

Reply:
I'm also looking for some guidance on the step- convention.

Reply:
It was a sass-calculated color, you're right that it's not part of the color palette. The "step-10" reflected `darker( primary, 10 )` in Sass. It was only meant to be temporary, a possible approach using Sass + custom properties, but I didn't clearly document it or remove it. `#1a5686` & `#0a4b78` are close enough to merge the two, and keep all properties set to colors from the palette.